src/ui/atoms/textField/TextField.tsx

```diff
@@ -53,12 +58,14 @@ export const TextField: React.FC<TextFieldProps> = ({
   value,
   defaultValue,
   helperText,
-  inputRef
+  inputRef,
+  fullWidth = false
 }: TextFieldProps): JSX.Element => {
   const inputClass = classNames(`okp4-text-field-core `, { error: hasError })
+  const containerClass = classNames(`okp4-text-field-main ${size}`, { 'full-width': fullWidth })
```

Comment:
You could use here shortcut syntax for object: 
```suggestion
  const containerClass = classNames(`okp4-text-field-main ${size}`, { fullWidth })
```

Reply:
My bad 😜